iree/compiler/Conversion/LinalgToSPIRV/Passes.h

```diff
@@ -90,8 +90,8 @@ void buildSPIRVTransformPassPipeline(OpPassManager &pm,
 // Patterns
 //===----------------------------------------------------------------------===//
 
-/// Populates patterns to tile and distribute linalg operations.
-void populateLinalgTileAndDistributePatterns(
+/// Populates patterns to tile and distribute linalg.copy operations.
+void populateTileAndDistributeLinalgCopyPatterns(
     MLIRContext *context, OwningRewritePatternList &patterns);
```

Comment:
Would it make sense to move this directly in VectorToGPU Or merge VectorToGPU with this file. Both should completely go away so it might not be worth it but it would get rid of this declaration.

Reply:
The handling of `linalg.copy` op is a bit intertwined between this file and the VectorToGPUPass. We essentially have two patterns for handling `linalg.copy`: `TileAndDistributeCopyOp` (used by VectorToGPUPass) and `SerializeAndDistributeCopyOp` (used by ConvertToGPUPass). They share common functionality. For now just leaving it mostly as-is; but I certainly agree with you we should future clean this up. Another step though.